lib/mosaic-objects/src/main/java/org/eclipse/mosaic/lib/objects/v2x/DuplicatableMessage.java

```diff
@@ -13,15 +13,13 @@
  * Contact: mosaic@fokus.fraunhofer.de
  */
 
-package org.eclipse.mosaic.lib.objects;
-
-import org.eclipse.mosaic.lib.objects.v2x.MessageRouting;
+package org.eclipse.mosaic.lib.objects.v2x;
 
 /**
- * Creates a copy of the message
- * @param <MessageT>
+ * A message that can duplicate itself.
+ * @param <T> a class that extends {@link org.eclipse.mosaic.lib.objects.v2x.V2xMessage}
```

Comment:
```suggestion
 * @param <T> a class that extends {@link V2xMessage}
```
As `DuplicatableMessage` is now in the same package you can remove the path I believe